dictionaries/mn_MN/package.json

```diff
@@ -42,7 +44,10 @@
   },
   "homepage": "https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/mn_MN#readme",
   "devDependencies": {
-    "dictionary-mn": "^3.0.0"
+    "dictionary-mn": "^3.0.1"
+  },
+  "dependencies": {
+    "node-fetch": "^3.0.0"
```

Comment:
If the files are being downloaded from the source, no need to have dependencies.

```suggestion
```

Note: `node-fetch` is deprecated and not needed. See: https://nodejs.org/en/learn/getting-started/fetch